directives/_static/js/aplus_tabs.js

```diff
@@ -0,0 +1,45 @@
+$(function() {
+
+    $('div.content-tabs').each(function() {
+        var contenttab_sel = $('<ul />', { class: "contenttab-selector" });
+        var i = 0;
+        
+        $('.tab-content', this).each(function() {
+            var sel_item = $('<li />', {
+                class: $(this).attr('id'),
+                text: $(this).find('.tab-title').text()
+            });
+            $(this).find('.tab-title').remove();
+            if (i++) {
+                $(this).hide();
+            } else {
+                sel_item.addClass('selected');
+            }
+            contenttab_sel.append(sel_item);
+            $(this).addClass('contenttab');
+        });
+
+        $('.tab-content', this).eq(0).before(contenttab_sel);
+        contenttab_sel = null;
+        i = null;
+    });
+
+
+    $('.contenttab-selector li').click(function(evt) {
+        evt.preventDefault();
+
+        var tabsblock = $(this).parents('.content-tabs');
+        
+
+        var sel_class = $(this).attr('class');
+        $('div.contenttab',tabsblock).hide();
+        $('div#' + sel_class,tabsblock).show();
+
+        $('ul.contenttab-selector li', tabsblock).removeClass('selected');
+        $('ul.contenttab-selector li.' + sel_class, tabsblock).addClass('selected');
+
+        sel_class = null;
+    });
```

Comment:
Is this tab component different from the similar component in Bootstrap3? It looks like you implemented this tab component without Bootstrap. Is that correct? Why did you decide to avoid BS?

One reason that I can guess is that we can't easily control the HTML output of the RST compilation and that HTML structure might be very different from the one expected by the Bootstrap API.

Reply:
Yes, I did not include any bootstrap here since I wanted to have an independent HTML element, able to work without relying on external CSS frameworks. I think it is better to keep the directives as independent as possible.